ArduSub/control_auto.cpp

```diff
@@ -463,10 +463,11 @@ void Sub::set_auto_yaw_roi(const Location &roi_location)
 #endif  // HAL_MOUNT_ENABLED
     } else {
 #if HAL_MOUNT_ENABLED
-        // check if mount type requires us to rotate the quad
+        // check if mount type requires us to rotate the sub
         if (!camera_mount.has_pan_control()) {
-            roi_WP = pv_location_to_vector(roi_location);
-            set_auto_yaw_mode(AUTO_YAW_ROI);
+            if(roi_location.get_vector_from_origin_NEU(roi_WP)) {
```

Comment:
nitpick: normally we have a space between the "if" and the bracket.

Reply:
done